src/main/java/duke/logic/parsers/ListCommandParser.java

```diff
@@ -17,15 +18,33 @@
      */
     @Override
     public ListCommand parse(String userInputStr) {
-        if (!userInputStr.isBlank()) {
-            try {
-                LocalDate date = LocalDate.parse(userInputStr,dateFormat);
-                return new ListCommand(date);
-            } catch (DateTimeParseException e) {
-                return new ListCommand(false, "Unable to parse \"" + userInputStr + "\" as a date.");
-            }
-        } else {
+        HashMap<String, String> argumentInfoMap;
+        LocalDate localDate = LocalDate.now();
+
+        if (userInputStr.isBlank()) {
+            System.out.println("Blank");
```

Comment:
You can probably remove the printing.

Reply:
Oh yeah I forgot. Thanks for pointing it out.